File: IOSApps/ObjectiveCApps/Yield Manager/Yield Manager/LoginRequest.m
//
//  LoginRequest.m
//  Yield Manager
//
//  Created by Praful Mantale on 1/23/15.
//  Copyright (c) 2015 prafulkumarmantale. All rights reserved.
//

#import "LoginRequest.h"
#import "Constants.h"

@implementation LoginRequest{
    
}

- (NSString *)getJSON{
    
    NSString *jsonStr =[NSString stringWithFormat:@"{\"%@\":\"%@\",\"%@\":\"%@\",\"%@\":\"%@\"}",keyLoginOrg,  self.organization, keyLoginUserName, self.userName, keyLoginPassword, self.password];
    
    
    return jsonStr;
}

@end

File: IOSApps/loginscreen/loginscreen/LoginRequest.swift
//
//  LoginRequest.swift
//  loginscreen
//
//  Created by Praful Mantale on 12/3/14.
//  Copyright (c) 2014 prafulkumarmantale. All rights reserved.
//

import Foundation

class LoginRequest{
    
    let orgKey : String = "org"
    let userKey : String = "user"
    let passwordKey : String = "pass"
    
    var orgName : String!
    var userName : String!
    var password : String!
    
    init(orgName : String, userName : String, password : String){
        self.orgName = orgName;
        self.userName = userName
        self.password = password
    }
    
    func getJsonString()->String{
        
        //"{\"org\":\"YMSBADemo\",\"user\":\"RWPraful\",\"pass\":\"Test123\"}";
        return "{\"\(orgKey)\":\"\(orgName)\",\"\(userKey)\":\"\(userName)\",\"\(passwordKey)\":\"\(password)\"}"
    }
    
}
